src/status_im/navigation/effects.cljs

```diff
@@ -199,15 +199,17 @@
   ([component] (show-overlay component {}))
   ([component opts]
    (navigation/dissmiss-overlay component)
-   (navigation/show-overlay
-    {:component {:name    component
-                 :id      component
-                 :options (merge (options/statusbar-and-navbar-options (:theme opts) nil nil)
-                                 {:layout  {:componentBackgroundColor :transparent
-                                            :orientation              ["portrait"]}
-                                  :overlay {:interceptTouchOutside true
-                                            :handleKeyboardEvents  true}}
-                                 opts)}})))
+   (let [theme         (:theme opts)
```

Comment:
thank you @jo-mut i've looked into show-overlay usages and I didn't find where we have :theme in opts, so probably it's better to remove it from code